model/atmosphere/diffusion/src/icon4py/model/atmosphere/diffusion/diffusion.py

```diff
@@ -348,7 +349,9 @@ class Diffusion:
     """Class that configures diffusion and does one diffusion step."""
 
     def __init__(
-        self, exchange: decomposition.ExchangeRuntime = decomposition.SingleNodeExchange()
+        self,
+        exchange: decomposition.ExchangeRuntime = decomposition.SingleNodeExchange(),
+        backend: Backend = gtx.gtfn_cpu,
```

Comment:
I though of assigning the backend to a variable analogous the exchange, but is probably is not even necessary...
```
self._backend = backend
```